src/BlazorUI/Bit.BlazorUI/Components/Utilities/Label/BitLabel.scss

```
@@ -14,13 +14,67 @@
+// Out of the page but still in the accessibility tree: a zero-sized box would be dropped from it by some screen
+// readers, so the label keeps a pixel of its own and is clipped away from it.
+.bit-lbl-vhd {
+    padding: 0;
+    width: 1px;
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    position: absolute;
+    white-space: nowrap;
+    clip-path: inset(50%);
+    clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)